Lib/test/test_exceptions.py

```diff
@@ -2195,6 +2195,22 @@ def test_range_of_offsets(self):
                     self.assertIn(expected, err.getvalue())
                     the_exception = exc
 
+    def test_subclass(self):
```

Comment:
Can we also add this test to main if is not there already?

Reply:
I created a PR: #127830